src/utils/api/calls.ts

```diff
@@ -19,7 +19,7 @@ export type PatientProps = {
   id?: string
   firstName: string
   lastName: string
-  birthdate: Date
+  birthdate: string
```

Comment:
we're storing date of birth as a string?

Reply:
To my knowledge they come in as a string from the API connection and need to be manually converted to dates